pyaedt/icepak.py

```diff
@@ -248,6 +248,47 @@ def create_source_block(self, object_name, input_power, assign_material=True, ma
             return bound
         return None
 
+    @aedt_exception_handler
+    def create_source_power(self, face_id, input_power="0W", thermal_condtion="Total Power", surface_heat="0irrad_W_per_m2", temperature="AmbientTemp", radiate=False, source_name=None):
+        """Create a source power for an face.
+
+        Parameters
+        ----------
+        face_id : int
+            Face Id
+        input_power : str or var
+            Input power.
+        surface_heat : str, optional
+            Surface Heat
+        temperature : str, optional
+            Temperature
+        radiate : bool
+            ``True`` if radiatinon enabled
+        source_name : str, optional
+            Source Name
+
+
+        Returns
+        -------
+        BoundaryObject
+            bound object or None
+
+        """
+        if not source_name:
+            source_name = generate_unique_name("Source")
+        props={}
+        props["Faces"] = [face_id]
+        props["Thermal Condition"] = thermal_condtion
+        props["Total Power"] = input_power
+        props["Surface Heat"] = surface_heat
+        props["Temperature"] = temperature
+        props["Radiation"] = OrderedDict({"Radiate" : radiate})
+        bound = BoundaryObject(self, source_name, props, 'SourceIcepak')
+        if bound.create():
+            self.boundaries.append(bound)
+            return bound
+        return None
```

Comment:
```suggestion
```
Any function or method without a `return` statement always returns `None`.  No need to have it here.